src/pywaterinfo/waterinfo.py

```diff
@@ -299,30 +306,44 @@ def _check_return_fields_format(self, return_fields, request="getTimeseriesValue
             )
 
     @staticmethod
-    def _parse_date(input_datetime):
+    def _parse_date(input_datetime, timezone=None):

```

Comment:
I think it is clearer to write timezone="UTC" as default and drop the if not timezone below.

Reply:
Good point, I adjusted as such and used a keyword default argument.